docs/tracingpolicy/README.md

```diff
@@ -0,0 +1,1193 @@
+# Tracing Policy
+
+`TracingPolicy` is a user-configurable Kubernetes custom resource (CR) that
+allows users to trace arbitrary events in the kernel and optionally define
+actions to take on a match, for enforcement for example. Currently, two types
+of events are supported: kprobes and tracepoints, but others may be added in
+the future (e.g., uprobes) by following a similar approach.
+
+Note that `TracingPolicy` can be considered low-level since they require
+knowledge about the Linux kernel and containers to be written correctly.
+
+For the complete custom resource definition (CRD) refer to the YAML file
+[`cilium.io_tracingpolicies.yaml`](https://github.com/cilium/tetragon/blob/main/pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml).
+One practical way to explore the CRD is to use `kubectl explain` against a
+Kubernetes API server on which it is installed, for example `kubectl explain
+tracingpolicy.spec.kprobes` provides field-specific documentation and details
+on kprobe spec.
+
+For bare metal or VM use cases without Kubernetes, the same YAML configuration
+can be passed via a flag to the Tetragon binary or via the `tetra` CLI to load
+the policies via gRPC.
```

Comment:
Can we make an example how to load it as a config file? I assume this is just an argument of the "--config-file" flag?

Reply:
yep https://github.com/cilium/tetragon/blob/745f6d2bab2ae98f40ec3c80a405a51e81dec38a/cmd/tetragon/flags.go#L40